packages/block-library/src/group/variations.js

```diff
@@ -10,12 +10,22 @@ const variations = [
 		title: __( 'Group' ),
 		description: __( 'Gather blocks in a container.' ),
 		attributes: { layout: { type: 'constrained' } },
-		scope: [ 'transform' ],
+		isDefault: true,
+		scope: [ 'inserter', 'transform' ],
+		isActive: ( blockAttributes ) =>
+			blockAttributes.layout?.type === 'constrained',
+		icon: group,
+	},
+	{
+		name: 'group-flow',
+		title: __( 'Group' ),
+		description: __( 'Gather blocks in a container.' ),
+		attributes: { layout: { type: 'default' } },
+		scope: [ 'block' ],
 		isActive: ( blockAttributes ) =>
 			! blockAttributes.layout ||
 			! blockAttributes.layout?.type ||
-			blockAttributes.layout?.type === 'default' ||
-			blockAttributes.layout?.type === 'constrained',
+			blockAttributes.layout?.type === 'default',
 		icon: group,
```

Comment:
Interesting. I didn't realise this approach had been taken before and there was already a group variation.

I wonder if this old variation can be removed? I'm not sure what a `scope` of `block` does, but any matching blocks would fallback to the normal block type, which would be fine.

Reply:
Good question, it seems to work well enough if I remove it. Let's try that.